### ISSUE TITLE
undocumented dependencies: coffeeify & brfs

### ISSUE THREAD

**Issue by avk** [evidenceprime/html-docx-js#42]
I had to npm install these browserify transforms before I could get your html-docx-js to be required:
`browserify: { transform: [ 'coffeeify', 'brfs' ] },`

**Comment by N1kto** [comment]
It would make sense to document them if those were `peerDependencies`, but they are `devDependencies`. Users  don't have to have them in order to be able to use the package.

They are needed for development only and are present in `devDependencies` section of `package.json` therefore. And even in this case there is no sense in documenting them because developers are free to use any bundling tool, should it be `webpack`, `browserify` or `rollup.js`, and any of these bundling tools has its one `coffee-scipt` compiler plugin and native methods polyfils plugins.

**Comment by avk** [comment]
@N1kto thanks, I did see them as `devDependencies` but I unfortunately had to add `coffeeify` and `bfrs` to my **package.json** before I could use the functionality. Maybe I'm missing something:

1. I'm using browserify. 
1. I included version 0.3.1 of `html-docx-js`.
1. In use:

```js
let htmlDocx = require('html-docx-js');
// and later...
htmlDocx.asBlob(html_doc);
```

Can anyone else reproduce on a browserify stack?
